frontend/src/app/updatesweb/mntner.service.ts

```diff
@@ -278,21 +277,6 @@ export class MntnerService {
         });
     }
 
-    public stripNccMntners(mntners: IMntByModel[], allowEmptyResult: boolean): IMntByModel[] {
```

Comment:
This logic is duplicated and allowEmptyResults is always "true" just "false" by tests